packages/hs-react-ui/src/components/Table/types.ts

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { StyledComponentBase } from 'styled-components';
+import { SubcomponentPropType } from '../commonTypes';
+
+export type ExpansionIconProps = {
+  isCollapsed: boolean;
+  onClick: any;
```

Comment:
Since you're touching them, would you mind cleaning up the onClicks and subcomponent types? At least make them match the StyledComponents, {} ... one so we can find and replace it later

Reply:
You got it.